geode-core/src/distributedTest/java/org/apache/geode/cache30/GlobalRegionCCEDUnitTest.java

```diff
@@ -163,78 +150,58 @@ public void testTombstones() {
+
+          // noinspection OverwrittenKey
           CCRegion.put("cckey0", "ccvalue");
+          // noinspection OverwrittenKey
           CCRegion.put("cckey0", "ccvalue"); // version number will end up at 4
         } catch (CacheException ex) {
           fail("While creating region", ex);
         }
       }
     };
-    vm0.invoke(createRegion);
-    vm1.invoke(createRegion);
-    vm1.invoke(new SerializableRunnable("Create local tombstone and adjust time") {
-      @Override
-      public void run() {
-        // make the entry for cckey0 a tombstone in this VM and set its
-        // modification time to be older than the tombstone GC interval. This
-        // means it could be in the process of being reaped by distributed-GC
-        RegionEntry entry = CCRegion.getRegionEntry("cckey0");
-        VersionTag tag = entry.getVersionStamp().asVersionTag();
-        assertTrue(tag.getEntryVersion() > 1);
-        tag.setVersionTimeStamp(
-            System.currentTimeMillis() - TombstoneService.REPLICATE_TOMBSTONE_TIMEOUT - 1000);
-        entry.getVersionStamp().setVersionTimeStamp(tag.getVersionTimeStamp());
-        try {
-          entry.makeTombstone(CCRegion, tag);
-        } catch (RegionClearedException e) {
-          fail("region was mysteriously cleared during unit testing", e);
-        }
+    vm0.invoke("Create Region", createRegion);
+    vm1.invoke("Create Region", createRegion);
+    vm1.invoke("Create local tombstone and adjust time", () -> {
+      // make the entry for cckey0 a tombstone in this VM and set its
+      // modification time to be older than the tombstone GC interval. This
+      // means it could be in the process of being reaped by distributed-GC
+      RegionEntry entry = CCRegion.getRegionEntry("cckey0");
+      VersionTag tag = entry.getVersionStamp().asVersionTag();
+      assertTrue(tag.getEntryVersion() > 1);
+      tag.setVersionTimeStamp(
+          System.currentTimeMillis() - TombstoneService.REPLICATE_TOMBSTONE_TIMEOUT - 1000);
+      entry.getVersionStamp().setVersionTimeStamp(tag.getVersionTimeStamp());
+      try {
+        entry.makeTombstone(CCRegion, tag);
+      } catch (RegionClearedException e) {
+        fail("region was mysteriously cleared during unit testing", e);
       }
     });
     // now remove the entry on vm0, simulating that it initiated a GC, and
     // perform a CREATE with a new version number
-    vm0.invoke(localDestroyRunnable);
-    vm1.invoke(new SerializableRunnable("Check that the create() was applied") {
-      @Override
-      public void run() {
-        RegionEntry entry = CCRegion.getRegionEntry("cckey0");
-        assertThat(entry.getVersionStamp().getEntryVersion()).isEqualTo(1);
+    vm0.invoke("Locally destroy the entry and do a create that will be propagated with v1", () -> {
+      CCRegion.getRegionMap().removeEntry("cckey0", CCRegion.getRegionEntry("cckey0"), true);
+      if (CCRegion.getRegionEntry("ckey0") != null) {
```

Comment:
Replace this if and fail block with an assertion:
```
assertThat(CCRegion.getRegionEntry("ckey0")).isNull();
```

Reply:
done